neqo-client/src/main.rs

```diff
@@ -432,7 +432,7 @@ fn process_loop(
             };
         }
         if !datagrams.is_empty() {
-            client.process_multiple_input(datagrams, Instant::now());
+            client.process_multiple_input(datagrams.iter(), Instant::now());
```

Comment:
I think that this is preferable here.

```suggestion
            client.process_multiple_input(&datagrams, Instant::now());
```

Reply:
See 3f76accc.